apps/system/js/ftu_ping.js

```diff
@@ -132,7 +133,8 @@
       function getSettings() {
         return new Promise(function(resolve, reject) {
           var allSettings = [FTU_PING_URL, FTU_PING_TRY_INTERVAL,
-                             FTU_PING_TIMEOUT, FTU_PING_MAX_NETWORK_FAILS].
+                             FTU_PING_TIMEOUT, FTU_PING_MAX_NETWORK_FAILS,
+                             FTU_PING_TELEMETRY_ENABLED_KEY].
```

Comment:
Nit: code would be more readable with each of the variables being on a separate line.